code/index.html

```diff
@@ -2,21 +2,130 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=7">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
       <!-- Some kind of logo and navbar -->
-    </header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy"> News</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">Contact</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">About</a></li>
+      </ul>
+    </nav>
+    <h1 class="site-name">STRAWBERRY NEWS <br> 
+     <span> Tastiest news of the week</span></h1>
+    
+    <img src="https://media.istockphoto.com/photos/strawberry-picture-id483079803?b=1&k=20&m=483079803&s=170667a&w=0&h=7zG9ttrpHETUZYihlF0hhI0kzPRoBfHCWKoq6zSrP7E=" 
+    class="logo" 
+    alt="logo"> 
+  </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <!-- The big news content -->
+  <section class="big-news">
+   <main>
```

Comment:
A suggestion is to wrap the main-tag around the whole content of the site; except the nav, header  & footer.